src/pages/docs/chat/setup.mdx

```diff
@@ -1,290 +0,0 @@
-  * Select the Ably Chat SDK for your target.
-
-To install the `ably-chat-swift` package in another Swift Package, add the following to your `Package.swift`:
-
-<Code>
-```swift
-.package(url: "https://github.com/ably/ably-chat-swift", from: "0.4.0"),
-```
-</Code>
-
-Import the SDK:
-
-<Code>
-```swift
-import AblyChat
-```
-</Code>
-</If>
-
-<If lang="kotlin">
-### Gradle <a id="gradle"/>
-
-The Ably Chat SDK is available on the Maven Central Repository. To include the dependency in your project, add the following to your `build.gradle.kts` file:
-
-<Code>
-```kotlin
-  implementation("com.ably.chat:chat-android:0.3.0")
-```
-</Code>
-
-For groovy:
-
-<Code>
-```kotlin
-  implementation 'com.ably.chat:chat-android:0.3.0'
-```
-</Code>
-</If>
-
-## Instantiate a client <a id="instantiate"/>
-
-Instantiate a realtime client using the Pub/Sub SDK and pass the generated client into the Chat constructor. <If lang="react">**It is strongly recommended that you initialise the clients outside of the React component tree** to avoid creating unnecessary additional connections to Ably caused by re-renders.</If>
-
-<If lang="react">
-Pass the `ChatClient` into the [`ChatClientProvider`](https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/functions/chat-react.ChatClientProvider.html). The `ChatClient` instance will be available to all child components in your React component tree.
-</If>
-
-<Code>
-```javascript
-import { LogLevel } from '@ably/chat'
-
-const realtimeClient = new Ably.Realtime({ key: '{{API_KEY}}', clientId: '<clientId>'});
-const chatClient = new ChatClient(realtimeClient, { logLevel: LogLevel.Error });
-```
-
-```react
-import { LogLevel } from '@ably/chat'
-
-const realtimeClient = new Ably.Realtime({ key: '{{API_KEY}}', clientId: '<clientId>'});
-const chatClient = new ChatClient(realtimeClient, { logLevel: LogLevel.Error });
-
-const App = () => {
-  return (
-    <ChatClientProvider client={chatClient}>
-      <RestOfYourApp />
-    </ChatClientProvider>
-  );
-};
-```
-
-```swift
-let realtimeOptions = ARTClientOptions()
-realtimeOptions.key = "{{API_KEY}}"
-realtimeOptions.clientId = "<clientId>"
-let realtime = ARTRealtime(options: realtimeOptions)
-let chatClient = DefaultChatClient(realtime: realtime, clientOptions: nil)
-```
-
-```kotlin
-import com.ably.chat.ChatClient
-import io.ably.lib.realtime.AblyRealtime
-import io.ably.lib.types.ClientOptions
-
-val realtimeClient = AblyRealtime(
-    ClientOptions().apply {
-        key = "{{API_KEY}}"
-        clientId = "<clientId>"
-    },
-)
-
-val chatClient = ChatClient(realtimeClient)
-```
-</Code>
-
-A [`ClientOptions`](/docs/api/realtime-sdk#client-options) object may be passed to the Pub/Sub SDK instance to further customize the connection, however at a minimum you must set an API key and provide a `clientId` to ensure that the client is [identified](/docs/auth/identified-clients).
```

Comment:
Similar comment here - we lose the info we have on logging for Chat, so we should move that elsewhere (maybe in connections?)